### DISCUSSION TITLE
Add Deepslate ores to grinder and IG recipes

### DISCUSSION THREAD

**Discussion by FV214Hardiel** [TechReborn/TechReborn#2571] [Show and tell]
Add Deepslate Ore blocks (like Deepslate Lapis or Deepslate Redstone Ores) to grinder recipes as their normal variants, They have same minecraft tag in vanilla (e.g. "tag": "minecraft:redstone_ores"), but in TR only normal variants have tag with 'c:' (e.g. "tag": "c:redstone_ores") that is used in machines recipes.

**Comment by equal-l2** [comment]
Apparently this has been fixed by 2fc6856, so will land on the release someday.
